build-system/tasks/prepend-global/index.js

```diff
@@ -62,24 +62,14 @@ function sanityCheck(str) {
  * @param {string} filename File containing the config
  * @param {boolean=} opt_localBranch Whether to use the local branch version
  * @param {string=} opt_branch If not the local branch, which branch to use
- * @return {!Promise}
+ * @return {!Promise<string>}
  */
-async function checkoutBranchConfigs(filename, opt_localBranch, opt_branch) {
+async function fetchConfigFromBranch_(filename, opt_localBranch, opt_branch) {
   if (opt_localBranch) {
-    return;
+    return fs.promises.readFile(filename, 'utf8');
   }
   const branch = opt_branch || 'origin/master';
-
-  try {
-    return await exec(`git checkout ${branch} ${filename}`);
-  } catch (e) {
-    // This means the files don't exist in master. Assume that it exists
-    // in the current branch.
-    if (/did not match any file/.test(e.message)) {
-      return;
-    }
```

Comment:
IIRC, all this code used to be necessary when we separately ran `gulp prepend-global` on dev machines as part of the old manual release workflow. Nice to see it all go away!